_posts/2019-02-13-extending-pg-search-sort.md

```diff
@@ -15,11 +15,14 @@ category: "development"
 excerpt: "Adjusting the sort capabilities pg_search to provide better search results."
 ---
 
-The search functionality of raywenderlich.com is powered by Postgresql using the `pg_search` ruby gem. Out of the box it is a fantastic library and makes building a search feature easy. Although, as easy as it was, we were quick to discover that our search results were not returning desired results. So I decided to take a closer look at how `pg_search` constructs its queries.
-
-## The default sort
-By default `pg_search` sorts results by a rank result of all the queried columns combined as a single value. In addition, you can set weights on each column so that certain columns will return a higher value than others. The configuration looks something like this:
-
+PostgreSQL comes out of the box with fantastic full text search capabilities. With a very robust feature set, it’s a popular option to implement search without having to use an external appliance like Elasticsearch or Solar. PostgreSQL is also the database behind [raywenderlich.com](https://www.raywenderlich.com/). Naturally it made sense to explore these features to build our very own [search page](https://raywenderlich.com/library). Fortunately there is a nice gem called [pg_search](https://github.com/Casecommons/pg_search) that leverages the search features in PostgreSQL and easily integrate them into a Rails app. It’s fairly simple to implement, once you’ve added it to your `Gemfile`, you can add full text search by including the module and calling the appropriate class methods The gem gives a very intuitive API that lets you take advantage of many of PostgreSQL’s search features.
```

Comment:
- www.raywenderlich.com/library
- Missing full-stop; "calling the appropriate class methods The gem..."